dataflow/src/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -397,12 +400,11 @@ public void visualizeBlockTransferInput(Block bb, Analysis<A, S, T> analysis) {
 
     @Override
     public void visualizeBlockNode(Node t, /*@Nullable*/ Analysis<A, S, T> analysis) {
-        A value = analysis.getValue(t);
-        String valueInfo = "";
-        if (value != null) {
-            valueInfo = "    > " + prepareString(value.toString());
+        this.sbBlock.append(prepareString(t.toString()) + "   [ " + prepareNodeType(t) + " ]");
+        A value = null;
+        if (analysis != null && (value = analysis.getValue(t)) != null) {
```

Comment:
Don't assign to a variable in a condition. Split this up into multiple conditions.